main.js

```diff
@@ -0,0 +1,13 @@
+let link = document.querySelector('li','ul')
+link.remove();
+
+let span = document.querySelector('span')
+span.remove();
+
+let posts = document.querySelector('.posts article')
+posts.remove();
```

Comment:
Although you're passing the tests for this one, you're not actually removing the **last** article as the readme.md asks. Instead, you're removing the **first** article.
```suggestion
let posts = document.querySelectorAll('.posts article')
let lastPost = posts[posts.length - 1];
lastPost.remove()
```
`querySelector` grabs the **first** matching element, NOT the **last** matching element. In order to grab the **last** matching element out of several matching elements, you need to use `querySelectorAll` to get a node list of **all** matching elements. Node lists are very similar to arrays, so you can grab the last element of a node list in the same way that you'd access the last element of an array: `let lastPost = posts[posts.length -1]`. Then you can delete _just_ the **last** article: `lastPost.remove()`